ui/src/views/mint/nfa-step/form-step/mint-form.context.ts

```diff
@@ -0,0 +1,54 @@
+import { FormField, useFormField } from '@/components';
+import { createContext, StringValidators } from '@/utils';
+import { useState } from 'react';
+
+/**
+ * The file size must be capped to a size that the contract can handle
+ */
+const DEFAULT_MAX_FILE_SIZE = 10; // in KB

```

Comment:
I think this is not needed now to follow the same pattern from the other validations.